### DISCUSSION TITLE
AWS Scan showing 2 High Priority Issues

### DISCUSSION THREAD

**Discussion by xs2tarunkukreja** [apache/airflow#37819] [Q&A]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### If "Other Airflow 2 version" selected, which one?

2.7.2

### What happened?

When we scan Airflow image through AWS Scan, it showing 2 HIGH priority issues -
CVE-2020-22218 - libssh2
CVE-2024-24806 - libuv1

### What you think should happen instead?

There should be no security issue.

### How to reproduce

Just scan airflow image with AWS Enhanced Scan

### Operating System

K8S 

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by eladkal** [comment]
Please read security policy https://github.com/apache/airflow?tab=security-ov-file
you will find:
> Specifically, we will ignore results of security scans that contain a list of dependencies of Airflow with dependencies in Airflow Docker reference image

**Reply by potiuk** [reply]
And specifically the chapter about images fixed at release time and what you should do: https://airflow.apache.org/docs/docker-stack/index.html#fixing-images-at-release-time if you wish to get rid of the vulnerabilties 

The best solution is to upgrade to latest released Airflow (currently 2.8.2) if you do not want to do it, you can build your own images - either way - the job to do is on your side in either case, you should just fix what's better for you @xs2tarunkukreja
